code/mob/player/player.dm

```diff
@@ -44,6 +44,10 @@
 		sdel(D)
 	playerEquipped = null
 
+	for(var/datum/O in playerSpellHolders)

```

Comment:
@DaedalusGame no I mean you literally remove the wrong one!
You didn't remove my copy of playerSpellHolders, you removed the one about statuseffects! haha